### ISSUE TITLE
Add guideline or custom branding/theme

### ISSUE THREAD

**Issue by sbunciak** [redhatcz/Thesis-management-system#86]

**Comment by jcechace** [comment]
Can you elaborate a bit on this? If this is meant as adding the support for custom themes, then I don't really think a single purpose system such as this should have this functionality (I would even consider it to be a sign of over-engineering)

**Comment by sbunciak** [comment]
This is not a single purposed system. It's open(-sourced) and thus *anyone* can use it. That means every user needs to have a way of modifying the look, preferably as easy as possible.

**Comment by jcechace** [comment]
The system was never designed to be a general purpose project management system. 

Nevertheless providing a custom style is only a matter of packaging the application with custom CSS. Support for custom layouts and such should be quite a low priority in my opinion. 

@VaclavDedik  What do you think?

**Comment by VaclavDedik** [comment]
I think it's quite a good idea. I got an email a while ago from a Romanian university asking how to use the system so I guess there might be someone who would appreciate such a feature.

Either way, I'd not focus on this right now, there is a backlog of quite a lot of issues that need to be taken care first. After the issues are resolved, we can maybe start discussing what we want to improve and maybe create a plan for a 2.0 version that could include this. So definitely a low priority atm.

**Comment by jcechace** [comment]
Agreed. Drafting some 2.0 plan and pushing custom branding towards this version makes sense.
